src/FAudio_gstreamer.c

```diff
@@ -241,7 +262,7 @@ static int FAudio_GSTREAMER_DecodeBlock(FAudioVoice *voice, FAudioBuffer *buffer
 	uint8_t *tmpBuf;
 	size_t tmpLen;
 
-	if (gstreamer->curBlock != ~0u && block != gstreamer->curBlock + 1)
+	if (gstreamer->curBlock != ~0u && block != gstreamer->curBlock && block != gstreamer->curBlock + 1)
```

Comment:
We're reasonably certain this is the breaking change... undoing this may fix WMA support?

Reply:
Still crashes after reverting that one line.